src/frontends/pytorch/src/op_table.cpp

```diff
@@ -438,6 +439,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::new_zeros", op::translate_new_zeros},
         {"aten::nonzero", op::translate_nonzero},
         {"aten::norm", op::translate_norm},
+        {"aten::normal_", op::translate_normal_},
```

Comment:
there is also normal without underscore
https://pytorch.org/docs/stable/generated/torch.normal.html#torch-normal
do we have plans to support it too?

Reply:
Makes sense to add it here, even though it is a pretty much different op. Added